routes/index.js

```diff
@@ -1,14 +1,23 @@
 var express = require('express');
 var router = express.Router();
 var Product = require('../models/product');
+var session = require('express-session');
 
 /* GET home page. */
 router.get('/', function(req, res, next) {
-  res.render('index', { products: /* ??? */ });
+  var products = Product.all;
+  var productObj = Product;
+  res.render('index', { products: products });
 });
 
 router.post('/cart', function(req, res, next) {
-  // How do we add an item to the cart?
+  var curSession = req.session.cart;
```

Comment:
You don't have to save this to another variable if you don't want to, you can use `req.session.cart` to save a line :)